.eslintrc.json

```diff
@@ -50,8 +50,24 @@
     "space-before-blocks": 2,
     "strict": [2, "global"],
     "vars-on-top": 2,
-    "yoda": [2, "never"]
+    "yoda": [2, "never"],
+    "sort-keys": ["error", "asc"]
   },
+  "overrides": [
+    {
+      "files": [
+        "test/**/*.js",
+        "features/**/*.js",
+        "features-sdk/**/*.js",
+        "lib/api/controllers/memoryStorage.js",
```

Comment:
I agree about overriding test files in eslint's configuration file, but I'm not sure about overrides for code files. Chances are we are going to forget about those pretty fast.

I think deactivating the sort-keys rule for these 3 files with a comment directive at the beginning of each file would be safer.